### PR TITLE
Add in-repo changelog to stonesense.

### PR DESCRIPTION
**Pull request by realSquidCoder**
See: https://github.com/DFHack/stonesense/issues/117

### CODE REVIEW

File: docs/changelog.txt [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+## Misc Improvements
+
+## Documentation
+
+## API
+
+## Lua
+
+## Removed
```
Comment (myk002) [inline docs/changelog.txt:41]:
```suggestion
## Fixes

## Misc Improvements

```

Comment (myk002) [reply docs/changelog.txt:41]: I suspect the template can be reduced to just those two headers. The other categories don't make much sense for the `stonesense` repo

File: historic-Changelog.txt
Comment (myk002) [inline historic-Changelog.txt:1]: I'd move this into the `docs` directory so it isn't more visible than the new "real" changelog.txt

I'd also recommend renaming it to `history.txt` to keep things short

File: docs/changelog.txt [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+## New Features
+
+## Fixes
+
+## Misc Improvements
+- `stonesense`: changed announcements to be right-aligned and limited it to only show the most recent 10 announcements.
+
+## Documentation
+- `stonesense`: added a changelog in the stonesense repo itself (stonesense/docs/changelog.txt).
```
Comment (myk002) [inline docs/changelog.txt:64]: changes that are not user-facing do not need to go in the changelog.  only things that would go in the public release notes.

File: docs/changelog.txt (hunk 1 above)
Comment (realSquidCoder) [reply docs/changelog.txt:41]: Think we should also leave Documentation and New Features?

File: historic-Changelog.txt
Comment (realSquidCoder) [reply historic-Changelog.txt:1]: Agreed, will change in next commit

File: docs/changelog.txt (hunk 2 above)
Comment (realSquidCoder) [reply docs/changelog.txt:64]: Additionally, is it not redundant to say stonesense? What should we do instead?

File: docs/changelog.txt (hunk 1 above)
Comment (myk002) [reply docs/changelog.txt]: `stonesense`'s docs are in the DFHack/dfhack repo. New Features could still be useful, depending on what you have a mind.

File: docs/changelog.txt (hunk 2 above)
Comment (myk002) [reply docs/changelog.txt]: `stonesense` is not as redundant as it seems. this file will get aggregated with the changelogs in the other repos and they all appear as a unified document in the release notes. We could potentially auto-inject a "`stonesense`" prefix, but that seems like more complexity than what is needed.

File: docs/changelog.txt [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+## New Features
+
+## Fixes
+
+## Misc Improvements
+- `stonesense`: changed announcements to be right-aligned and limited it to only show the most recent 10 announcements.
+
+## Removed
+- `stonesense`: old changelog has been deprecated and renamed `history.txt` and moved to docs folder.
```
Comment (myk002) [inline docs/changelog.txt:54]:
```suggestion
```

Comment (myk002) [reply docs/changelog.txt:54]: won't be user-visible

**Review by myk002 — APPROVED** [review]

**Pull request merged**